File: code/chapter-03/guess_game_v2.py
print("Welcome! Let's play the number guessing game.\nVersion 2\n")

secret_number = "7"
guess = None

# Game loop
while guess != secret_number:
    guess = input("Guess a number from 1 to 10? ")

    if guess == secret_number:
        print("You guessed it!")
    else:
        print("Wrong! Guess again.")

print("Goodbye.")

File: code/chapter-03/guess_game_v2.js
console.log("Welcome! Let's play the number guessing game.\nVersion 2\n");

let secret_number = "7";
let guess = null;

// Game loop
while (guess !== secret_number) {
    // Reassign guess to the user's input
    guess = await input("Guess a number from 1 to 10?");
    
    if (guess === secret_number) {
        console.log("You guessed it!");
    } else {
        console.log("Wrong! Guess again.");
    }
}

console.log("Goodbye.");
